lib/utils.re

```diff
@@ -164,8 +145,57 @@ let checkAcceptable = (classNames, loc, tailwindFile) => {
   List.iter(isAcceptable, classNames);
 };
 
-let validate = (~classNames, ~loc, ~tailwindFile) => {
+type cachedAcceptableClassNames = {
+  tailwindCssHash: string,
+  acceptableClassNames: StringSet.t,
+};
+
+let getCachedAcceptaleClassNames = (~filename, ~tailwindFileContent) =>
+  switch (Pervasives.open_in(filename)) {
+  | input =>
+    print_endline("read from file cached");
+    let fileContent: cachedAcceptableClassNames =
+      Pervasives.input_value(input);
+    Pervasives.close_in(input);
+    if (Digest.string(tailwindFileContent) == fileContent.tailwindCssHash) {
+      Some(fileContent.acceptableClassNames);
+    } else {
+      None;
+    };
+  | exception _ => None
+  };
+
+let acceptableClassNames = ref(None);
+let validate = (~classNames, ~loc, ~tailwindFileContent) => {
+  let filename = "activeClassNames.bin";
```

Comment:
**Question:** I want to confirm, is this the file name of `tailwind-ppx`'s cache? If possible, I'd prefer to follow the pattern set by [graphql-ppx](https://github.com/reasonml-community/graphql_ppx#ignore-graphql_ppx_cache-in-your-version-control) and name the file something like `.tailwind_ppx_cache`